### PR TITLE
Hook up the command again (#9587)

### PR DESCRIPTION
**Pull request by aaomidi**
Issue #9585

### CODE REVIEW

File: extensions/azurecore/src/account-provider/azureAccountProviderService.ts [hunk 1]
```diff
@@ -70,8 +70,7 @@ export class AzureAccountProviderService implements vscode.Disposable {
 		// let self = this;
```
Comment (Charles-Gagnon) [inline extensions/azurecore/src/account-provider/azureAccountProviderService.ts]: While you're here mind cleaning this up too?

Comment (Charles-Gagnon) [reply extensions/azurecore/src/account-provider/azureAccountProviderService.ts]: Oh this is a port request. Doesn't really matter for this then.

Comment (aaomidi) [reply extensions/azurecore/src/account-provider/azureAccountProviderService.ts]: I'll clean up in my MSAL work

**Comment by aaomidi** [comment]
Merging PR based on approval from @kburtram

**Pull request merged**